frontend/src/scenes/surveys/surveyLogic.tsx

```diff
@@ -82,10 +84,30 @@ export const getSurveyDataQuery = (surveyName: string): DataTableNode => {
     return surveyDataQuery
 }
 
+export const getSurveyMetricsQueries = (surveyId: string, surveyName: string): SurveyMetricsQueries => {
+    const surveysShownHogqlQuery = `select count() as 'survey shown' from events where event == '${surveyName} survey shown' and properties.$survey_id == '${surveyId}'`
+    const surveysDismissedHogqlQuery = `select count() as 'survey dismissed' from events where event == '${surveyName} survey dismissed' and properties.$survey_id == '${surveyId}'`
```

Comment:
I'd strongly recommend not relying on survey names in events.

Not only is this prone to errors (whitespace and otherwise), but also, say if you wanted to allow users to change the name of the survey in the future (because say, they input an incorrect name, or want to clarify something). This becomes impossibly hard with our current setup since everything relies on the name.

I'd recommend using the survey IDs for all filtering of responses + counts.

Reply:
sure, i can just rely on the survey id property that gets sent then. For readability though, I think users will generally want to just have the survey name + some format of "shown/dismissed" in their event name.